fedimint-core/src/api.rs

```diff
@@ -1099,7 +1100,11 @@ mod tests {
         };
 
         let bech32 = connect.to_string();
+        let json = serde_json::to_string(&connect).unwrap();
+        assert_eq!(json, format!("\"{bech32}\""));
```

Comment:
json and bech32 being interwined just make reading harder.

I'm fine with one test, but first render and compare bech32, then render and compare json (or the other way around). Just not two at the same time.